README.md

```diff
@@ -8,7 +8,9 @@ npm run start
 
 ## Requirements and Configuration
 
-You’ll need to have Node 8.16.0 or Node 10.16.0 or later version on your local development machine
+You’ll need to have Node v14.19.0 on your local development machine
```

Comment:
We can set this as the engine in package.json so it will be checked automatically
https://docs.npmjs.com/cli/v8/configuring-npm/package-json#engines

Reply:
Do you guys think it makes sense to add `.nvmrc` as well?

Reply:
The docker file says node 12.19.0 so we should update this to be greater then 12

Reply:
gonna open new PR for node version updates. 
In current PR would be just package fixes